src/components/state-history-chart-line.html

```diff
@@ -187,20 +186,30 @@
         if (domain === 'thermostat' || domain === 'climate') {
           // We differentiate between thermostats that have a target temperature
           // range versus ones that have just a target temperature
-          hasTargetRange = states.states.some(state => state.attributes &&
+          const hasTargetRange = states.states.some(state => state.attributes &&
             state.attributes.target_temp_high !== state.attributes.target_temp_low);
+          const hasHeat = states.states.some(state => state.state === 'heat');
```

Comment:
I also though of 2 different approaches:
1) Go over all states except 'off' and 'idle' and create a separate columns for each.
2) Create a single "on" column for all states except 'off' and 'idle'.